subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CheckstyleReports.java

```diff
@@ -34,7 +34,7 @@
      * @return The checkstyle HTML report
      * @since 2.10
      */
-    SingleFileReport getHtml();
+    CheckstyleHtmlReport getHtml();
```

Comment:
Why change this? This file doesn't use any of the new methods in the CheckstyleHtmlReport, right?